torch/csrc/jit/codegen/cuda/ir_iostream.h

```diff
@@ -47,6 +47,7 @@ struct Add;
  */
 
 struct TORCH_CUDA_API IRPrinter : public OptInConstDispatch {
+ public:
```

Comment:
Point it out, just in case this is a temporary debug thing that we forgot to move back.